.gitleaksignore

```diff
@@ -1,3 +1,4 @@
 # SEE: https://github.com/gitleaks/gitleaks/blob/master/README.md#gitleaksignore
 
 cd9c0efec38c5d63053dd865e5d4e207c0760d91:docs/guides/Perform_static_analysis.md:generic-api-key:37
+src/__tests__/components/molecules/LoginStatus.test.tsx:jwt:23
```

Comment:
???

Reply:
I was getting a scan secret error on the pipeline. @chris-elliott-nhsd has added a token/key that is used for tests only on his branch.